lib/deploy/docker.js

```diff
@@ -12,20 +13,93 @@ exports.dockerize = () => {
 
   setTimeout(() => {
     configFileExists();
-
-    // currently works on linux only
-    if(os.type() === 'Linux') {
-
-      console.log('\n');
-      shell.exec('sudo docker-compose up', (err) => {
-        if(err) {
-          console.log(chalk.red.bold('something went wrong!'));
-          process.exit(1);
-        } else {
-          console.log('You are all set...!\nserver:- localhost:9000\nclient:- localhost:8080');
+    
+    //check if docker is installed on your system and proceed with the installation depending on the os
+    if(!shell.which('docker')){
+      shell.echo('Sorry, docker is not installed on your system, Do you want to install docker?');
+      inquirer.prompt([{
```

Comment:
Instead of asking the user to type in `Y/N` it is good to have some sort of confirmation as provided by inquirer via `type:confirm` property.

Reply:
@jamesgeorge007 okay. That'd be better indeed.

Reply:
I have made the changes. I will push them in sometime